assets/sass/breakpoints/_helper.scss

```diff
@@ -0,0 +1,3 @@
+@forward "./_mobileView.scss";
+@forward "./_tabletView.scss";
+@forward "./_desktopView.scss";
```

Comment:
Ovo nije dobra arhitektura jer se css redefinise u zavisnosti od mediaQuerija. Bolji nacin je da se klasa/selektor definise samo jednom a u okviru njega hendla responsive:
`body {
    display: flex;
    text-align: center;
    flex-direction: column;
    ...
    
    @media screen and (min-width: 700px) {...}
    @media screen and (min-width: 1300px) {...}
    ...
}`